contracts/payloads/common/interfaces/ILiteSigs.sol

```diff
@@ -72,6 +72,8 @@ interface ILiteSigs {
         string memory swapCallData_
```

Comment:
@xma8lu We need to change this as well, right?

Reply:
checking it